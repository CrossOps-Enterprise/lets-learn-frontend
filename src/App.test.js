import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

it('should render App component without crashing', () => {
  const component = shallow(<App />)
  expect(component).toMatchSnapshot()
})

it('App component should have the text Hello World', () => {
  const component = shallow(<App />)
  expect(component.text()).toEqual('Hello World')
})
