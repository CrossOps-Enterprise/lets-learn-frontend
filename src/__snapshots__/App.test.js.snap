// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component without crashing 1`] = `
<div>
  Hello World
</div>
`;
